File: Step5-Python/4344.py
n=int(input()) #테스트 케이스 개수

for i in range(n):
    num=list(map(int,input().split()))
    avg=sum(num[1:])/num[0] #평균

    count=0
    for j in num[1:]:
        if j>avg:
            count+=1
    
    rate=count/num[0]*100
    print("{:.3f}%".format(rate))

File: Step5-C/4344.c
#define _CRT_SECURE_NO_WARNINGS
#include <stdio.h>

int main()
{
	int num;
	scanf("%d", &num);

	for (int i = 0; i < num; i++)
	{
		int people;
		scanf("%d", &people);

		int score[1001];
		int sum = 0;
		for (int j = 0; j < people; j++)
		{
			scanf("%d", &score[j]);
			sum += score[j];
		}
		float avg = (float)sum / people;

		int count = 0;
		for (int k = 0; k < people; k++)
		{
			if (score[k] > avg)
				count += 1;
		}

		float rate = (float)count / people * 100;
		printf("%.3f%%\n", rate);
		
	}
	return 0;
}